packages/zoe/src/contractFacet/exit.js

```diff
@@ -26,12 +26,22 @@ export const makeExitObj = (proposal, zoeSeatAdmin, zcfSeatAdmin) => {
 
   if (exitKind === 'afterDeadline') {
     // Automatically exit the seat after deadline.
-    E(proposal.exit.afterDeadline.timer).setWakeup(
-      proposal.exit.afterDeadline.deadline,
-      harden({
-        wake: exitFn,
-      }),
-    );
+    E(proposal.exit.afterDeadline.timer)
+      .setWakeup(
+        proposal.exit.afterDeadline.deadline,
+        harden({
+          wake: exitFn,
+        }),
+      )
+      .catch(err => {
```

Comment:
For errors that manifest as a rejected promise's rejection reason, prefer `reason` to `err`, but either is acceptable.

Reply:
Ok, sure, that sounds good. I'll change it.